cmd/vela-git/plugin.go

```diff
@@ -88,6 +88,20 @@ func (p *Plugin) Exec() error {
 		}
 	}
 
+	// check if it's a pull request
+	if p.Repo.PrTargetBranch != "" {
```

Comment:
We'd augment this slightly to instead rely on the `bool` opt-in parameter we created previously.

Something like:

```go
if p.Repo.FetchTarget {
```